public/API/API_GetGameExtended.php

```diff
@@ -81,7 +82,8 @@
     'Publisher' => $game->Publisher,
     'Developer' => $game->Developer,
     'Genre' => $game->Genre,
-    'Released' => $game->Released,
+    'Released' => $game->released_at?->format('Y-m-d H:i:s'),
```

Comment:
Looking at the tests below, the format for `Released` used to be: `'M Y'`. This is _technically_ also a breaking change.

Reply:
That was the format in the tests, but not in real life. Up until now, the field has been completely free-form text. Here's a few examples that are now hidden as of the latest release:

```
109,Oktober 1987
183,"Novermber 15, 1996 (UK)"
524,Never
542,Physically
626,"JP:July 16, 1993, NA:October 1993, EU:1993"
874,"JP:November 27, 1992 NA:December 1, 1993"
910,Nov 15th 2021 - Dec 15th 2021
```

Reply:
Cool, then this is much better.